hcx-apis/src/main/java/org/swasth/hcx/controllers/v1/ParticipantController.java

```diff
@@ -113,4 +111,12 @@ private ParticipantResponse errorResponse(ErrorCodes code, String message, Throw
         return resp;
     }
 
+    private HttpEntity<ParticipantResponse> validateParticipant (Map<String, Object> body) {
```

Comment:
Update the method to validateEndpointUrl

Reply:
done